config.json

```
@@ -2218,6 +2218,23 @@
         ],
         "difficulty": 5
       },
+      {
+        "slug": "state-of-tic-tac-toe",
+        "name": "State of Tic Tac Toe",
+        "uuid": "905ac84d-7ba1-4208-bee1-55d8efde0c01",
+        "prerequisites": [
+          "strings",
+          "tuples",
+          "atoms",
+          "cond",
+          "lists",
+          "enum",
+          "maps",
+          "list-comprehensions"
+        ],
```

My solution included a `case`, and yours included a charlist:
```suggestion
        "prerequisites": [
          "strings",
          "tuples",
          "atoms",
          "cond",
          "lists",
          "enum",
          "maps",
          "list-comprehensions",
          "case",
          "charlists"
        ],
```